src/main/java/com/jwt/jwt/Entity/UserData.java

```diff
@@ -0,0 +1,38 @@
+package com.jwt.jwt.Entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+import org.springframework.security.core.GrantedAuthority;
+
+import java.io.Serial;
+import java.io.Serializable;
+
+@Entity
+@Table(name = "user_data")
+@Data
+public class UserData implements GrantedAuthority {
+
+    @Id
+    @GeneratedValue
+    private Integer id;
+    private String username = "";
+    private String password = "";
+    private String role = "";
```

Comment:
Please use enum to define roles.
Please make sure that you address the comments added on Parth and Vaibhav's PR too.